src/views/Swap/Swap.vue

```diff
@@ -911,8 +900,8 @@ export default {
       }
       if (
         this.selectedQuote?.receiveFee &&
-        BN(this.selectedQuote.toAmount).lt(
-          BN(this.selectedQuote.receiveFee).times(this.selectedQuote?.maxFeeSlippageMultiplier || 1)
+        BN(this.selectedQuote?.toAmount).lt(
```

Comment:
`this.selectedQuote` is checked in the first condition
`this.selectedQuote?.receiveFee` so no need to do it after the && statement

Reply:
fixed @kraikov